pkg/apis/pingcap/v1alpha1/types.go

```diff
@@ -1356,6 +1371,14 @@ type BackupSpec struct {
 	ServiceAccount string `json:"serviceAccount,omitempty"`
 	// CleanPolicy denotes whether to clean backup data when the object is deleted from the cluster, if not set, the backup data will be retained
 	CleanPolicy CleanPolicyType `json:"cleanPolicy,omitempty"`
+
```

Comment:
Please also add annotations and labels for the [backupschedule](https://github.com/pingcap/tidb-operator/blob/master/pkg/apis/pingcap/v1alpha1/types.go#L1480) spec, the Operator will create backup CRs according to the `BackupTemplate` field in the backupSchedule CR, so the labels and annotations should also inherit from the backupSchedule CR and may also overwrite them.

Reply:
Where are the code changes for the Backup/Restore/BackupSchedule?